lib/openapi_parser/schemas/operation.rb

```diff
@@ -30,5 +30,21 @@ def validate_request_body(content_type, params, options)
     def validate_response(response_body, response_validate_options)
       responses&.validate(response_body, response_validate_options)
     end
+
+    def validate_security_schemes(securitySchemes, headers)
+      securitySchemes&.each do |securityScheme|
```

Comment:
your english is ok. I have to understand, `security requirement object` does not have schema information about `type`, `scheme` and `bearerFormat`

Reply:
@ota42y Hi. any updates? I'm thinking of doing my own fork because i cannot find the solution with:

```ruby
def validate_security_schemes(securitySchemes, headers)
  validate_results = security.map do |s| # s is security requirement object
    # check all security
    s.validate_security_schemes(securitySchemes, headers) <-- this needs to be object
  end
  if validate_results.count(true) == 1
    return true # accept 
  end
  return ValidateSecurityError()
end
```

Reply:
```
security requirement object does not have schema information about type, scheme and bearerFormat is only in SecuritySchema object.
```
You are right. I confused security scheme object and secuirty requirement object 🙇